dll/cpl/input/settings_page.c

```diff
@@ -405,7 +432,20 @@ OnNotifySettingsPage(HWND hwndDlg, LPARAM lParam)
         case PSN_APPLY:
         {
             /* Write Input Methods list to registry */
-            InputList_Process();
+            if (InputList_Process())
```

Comment:
So I understand there is no other way to make win32k understand the font substitutes without rebooting?

Reply:
There are some ways to notify the specific status. Registry key or value, file, folder, or DDE share.